### PR TITLE
Add details of the GitHub Action README.md file for the samples browser

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Add details of the GitHub Action README.md file for the samples browser

Related to #dotnet/docs#23117

### CODE REVIEW

File: github-actions/DotNet.GitHubAction/README.md [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+urlFragment: "create-dotnet-github-action"
+description: "A .NET 5 console application that contains the example from Create a GitHub Action with .NET tutorial."
+---
 
-    TODO:
+# Create a GitHub Action with .NET
 
-    1. Write the tutorial around this code in dotnet/docs
-    2. Write the README.md file content to light up the samples/browser experience
+This sample is a console application written in C# from the [Create a GitHub Action with .NETtutorial](https://docs.microsoft.com/dotnet/github/create-dotnet-github-action). The article gives an overview of what is required to create a .NET GitHub action.
```
Comment (BillWagner) [inline github-actions/DotNet.GitHubAction/README.md:15]: I think this folder will change. We discussed dotnet/fundamentals/devops as a new folder.

**Review by BillWagner — APPROVED** [review]
This LGTM @IEvangelist 

I had two comments, and then it's ready.

File: github-actions/DotNet.GitHubAction/README.md (hunk 1 above)
Comment (IEvangelist) [inline github-actions/DotNet.GitHubAction/README.md:15]:
```suggestion
This sample is a console application written in C# from the [Create a GitHub Action with .NETtutorial](https://docs.microsoft.com/dotnet/devops/create-dotnet-github-action). The article gives an overview of what is required to create a .NET GitHub action.
```
```suggestion
This sample is a console application written in C# from the [Create a GitHub Action with .NETtutorial](https://docs.microsoft.com/dotnet/github/create-dotnet-github-action). The article gives an overview of what is required to create a .NET GitHub action.
```

File: github-actions/DotNet.GitHubAction/README.md [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+    1. On the menu bar, choose **File** > **Open** > **Project/Solution**.
+    2. Navigate to the folder that holds the unzipped sample code, and open the C# project (.csproj) file.
+    3. Choose the <kbd>F5</kbd> key to run with debugging, or <kbd>Ctrl</kbd>+<kbd>F5</kbd> keys to run the project without debugging.
+3. From the command line:
+   1. Navigate to the folder that holds the unzipped sample code.
+   2. At the command line, type `dotnet run`.
+
+## More information
+
+- <https://docs.microsoft.com/dotnet/github/create-dotnet-github-action>
```
Comment (IEvangelist) [inline github-actions/DotNet.GitHubAction/README.md:36]:
```suggestion
- <https://docs.microsoft.com/dotnet/devops/create-dotnet-github-action>
```
```suggestion
- <https://docs.microsoft.com/dotnet/github/create-dotnet-github-action>
```

**Pull request merged**